l10n_it_ddt/models/stock.py

```diff
@@ -20,6 +21,22 @@ class StockPicking(models.Model):
         string='DdT',
         copy=False, )
 
+    @api.multi
+    @api.constrains('state')
+    def _ddt_constrain_state(self):
+        """Prevent pickings that are included in a DDT to be set as done
+        unless it is being processed by DDT"""
+        if self.env.context.get('ddt_processing', False):
+            return
+        for picking in self:
+            if picking.state == 'done' and picking.ddt_ids:
+                raise ValidationError(
+                    _('Cannot set to "Done" picking %s '
+                      'because it is included in DDT %s.\n'
+                      'Process the DDT or unlink the picking from the DDT.')
```

Comment:
@SimoRubi questo non è un caso d'uso da escludere dal modulo. L'utente deve poter processare i picking indipendentemente dal DDT

Reply:
Fatto in https://github.com/OCA/l10n-italy/pull/656/commits/d4f7bd8ce7b584d6e387e4385d86b0d55bf5a946